postprocessing/mppnccombine/mppnccombine.c

```diff
@@ -24,6 +24,10 @@
                  programming interface "mpp_io_mod"
                  (http://www.gfdl.noaa.gov/~vb/mpp_io.html) by V. Balaji.
 
+  V2.2.8: Hans.Vahlenkamp@noaa.gov
+          If no netCDF format is specified (no -64 or -n4 options) for the
+          output file then use the netCDF format of the first input file;
+          use netCDF3 64-bit offset format or netCDF4 classic format.
```

Comment:
Can the comment be a bit more explicit and say "users have the choice to request..."?

Reply:
Ok, I reworded the version comment and the usage message a bit.